freqtrade/enums/liqformula.py

```diff
@@ -0,0 +1,108 @@
+# from decimal import Decimal
+from enum import Enum
+
+from freqtrade.enums.collateral import Collateral
+from freqtrade.enums.tradingmode import TradingMode
+from freqtrade.exceptions import OperationalException
+
+
+# from math import ceil
+
+
+class LiqFormula(Enum):
+    """Equations to calculate liquidation price"""
+
+    BINANCE = "Binance"
+    KRAKEN = "Kraken"
+    FTX = "FTX"
+    NONE = None
+
+    def __call__(self, **k):
+
+        trading_mode: TradingMode = k['trading_mode']
+        if trading_mode == TradingMode.SPOT or self.name == "NONE":
+            return None
+
+        collateral: Collateral = k['collateral']
+
+        if self.name == "BINANCE":
+            return binance(trading_mode, collateral)
+        elif self.name == "KRAKEN":
+            return kraken(trading_mode, collateral)
+        elif self.name == "FTX":
+            return ftx(trading_mode, collateral)
+        else:
+            exception(self.name, trading_mode, collateral)
```

Comment:
As mentioned in some other PR - please remove all logic from the `freqtrade.enum` package.

Again - i don't fully understand the difference between this and #5348.